src/App.tsx

```diff
@@ -0,0 +1,49 @@
+import useCalculator from 'src/hooks/useCalculator.ts';
+
+function App() {
+	const { enter, calculate, clear, display } = useCalculator();
+
+	const digits = ['9', '8', '7', '6', '5', '4', '3', '2', '1', '0'];
+
+	const operators = ['/', 'x', '-', '+'];
+
+	return (
+		<div className="calculator">
+			<h1 id="total">{display}</h1>
+			<div className="digits flex">
+				{digits.map(digit => (
+					<button className="digit" key={digit} onClick={() => enter(digit)}>
+						{digit}
+					</button>
+				))}
+			</div>
+			<div className="modifiers subgrid">
+				<button
+					className="modifier"
+					onClick={() => {
+						clear();
+					}}
```

Comment:
아래처럼만 작성해도 동작할거 같은데 익명함수여야 하는 이유가 있었는지 궁금하네용 ㅎㅎ

```suggestion
					onClick={clear}
```

Reply:
처음 커스텀 훅을 구현할때 "컴포넌트에서 전달받은 함수들(enter, caculate, clear)을 실행한다" 라고 생각하였습니다. 그래서 AC 클릭시 clear를 실행시키기 위해 익명함수로 구현하였습니다.

클릭 말고도 다른 경우에도 clear를 실행시킬 수 있다고 생각했기 때문입니다.

다시 생각해보니 가독성 측면에서 익명함수를 사용하지 않는 것이 더 좋을 것 같긴하네요.

다음 미션때는 이런 부분을 더 생각해보면서 구현해봐야 할 것 같아요🥲